content/docs/2.0/deploy.md

```diff
@@ -8,6 +8,8 @@ We provide a few approaches to deploy KEDA runtime in your Kubernetes clusters:
 - [Operator Hub](#operatorhub)
 - [YAML declarations](#yaml)
 
+> 💡 **NOTE:** KEDA requires Kubernetes cluster version 1.16 and higher
```

Comment:
Should we add this to FAQ as well?

Reply:
Great idea, I am gonna add that

Reply:
https://github.com/kedacore/keda-docs/pull/273

Reply:
this needs to be solved first^ (versioned FAQ)

Reply:
@tomkerkhove added info to the FAQ, PTAL